### PR TITLE
add prometheus metrics timestamp

### PR DESCRIPTION
**Pull request by Betula-L**
This PR is to fix https://github.com/google/cadvisor/issues/2123

It includes,

- Update vendor prometheus client_golang 
- Add exact scrape timestamp for each Prometheus counter metric

### CODE REVIEW

**Comment by Betula-L** [comment]
I signed it!

**Comment by dashpole** [comment]
looks like you will need to update the test data to include timestamps as well: https://github.com/google/cadvisor/tree/master/metrics/testdata

**Comment by Betula-L** [comment]
I forgot test. Finished now.

File: metrics/testdata/prometheus_metrics [hunk 1]
```diff
@@ -224,32 +224,11 @@ container_spec_cpu_shares{container_env_foo_env="prod",container_label_foo_label
...
 # HELP container_tasks_state Number of tasks in given state
 # TYPE container_tasks_state gauge
-container_tasks_state{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",state="iowaiting",zone_name="hello"} 54
-container_tasks_state{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",state="running",zone_name="hello"} 51
-container_tasks_state{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",state="sleeping",zone_name="hello"} 50
-container_tasks_state{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",state="stopped",zone_name="hello"} 52
-container_tasks_state{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",state="uninterruptible",zone_name="hello"} 53
-# HELP http_request_duration_microseconds The HTTP request latencies in microseconds.
```
Comment (dashpole) [inline metrics/testdata/prometheus_metrics]: Why did these metrics disappear?

**Pull request merged**